examples/ros-service-call-module/src/App.css

```diff
@@ -0,0 +1,92 @@
+.App {
+ min-height: 100vh;
+ width: 100vw;
+ display: flex;
+ align-items: center;
+ justify-content: center;
```

Comment:
not a terrible deal breaker, but careful of indentation here (looks like its supposed to be 2-spaces)